dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -267,4 +267,22 @@ def stub_country(code)
     assert_equal false, app_options[:level]['submittable']
   end
 
+  test 'Milestone posting is disabled if Gatekeeper flag is set' do
```

Comment:
Gatekeeper and DCDO state should automatically get cleared by the test helper.  If that's not the case let me know and I'll fix it.